pyproject.toml

```diff
@@ -51,15 +51,19 @@ scipy = [
 matplotlib = ">=3.6.0"
 pyyaml = ">=5.3"
 defusedxml = "^0.7.1"
+pillow = ">=9.4"
 opencv-python = { version = ">=4.5.5.64", optional = true }
 opencv-python-headless = ">=4.5.5.64"
 requests = { version = ">=2.26.0,<=2.32.3", optional = true }
 tqdm = { version = ">=4.62.3,<=4.66.5", optional = true }
-pillow = ">=9.4"
+# pandas: picked lowest major version that supports Python 3.8
+pandas = { version = ">=2.0.0", optional = true }
```

Comment:
@onuralpszr, do we need to do something extra here?

Reply:
Sorry, I miss this message because was hidden, making optional to add is correct I also see "stub" package which also optional also good.  If I may I would like to also say If you need faster metrics and better RAM management, We should strongly consider "polars" as well.  I can give you raw example and send you colab example what I meant about it.  Plus we can convert to pandas as well.  or read from pandas,numpy etc as well. So cards are still open. You saw speed of "ruff" (rust) you can see this one too.

Reply:
Doc references:

https://docs.pola.rs/api/python/stable/reference/functions.html
https://docs.pola.rs/api/python/stable/reference/dataframe/export.html

I can show more and explain as well.